src/main/java/org/spongepowered/api/data/Keys.java

```diff
@@ -788,9 +788,9 @@
     public static final Supplier<Key<BoundedValue<Integer>>> FIRE_TICKS = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "FIRE_TICKS");
 
     /**
-     * The time a {@link Player} first played on the Server.
+     * The time a {@link Player} first joined on the Server.
```

Comment:
{@link Player} -> {@link ServerPlayer player}

Reply:
Actually, this should be a User